sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 12.5.0-beta.1 (2021-02-09)
+
+- Now support Batch operations scoped to the Container level. You can use `ContainerClient.getBlobBatchClient()` to get such kind of `BlobBatchClient`. The `ContainerClient` can be authenticated via a Container-level SAS.
```

Comment:
>Container-level SAS [](start = 201, length = 19)

why container level SAS is specifically mentioned here?

Reply:
Removed.